k8s/jenkins/jenkins-server.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: jenkins
-  namespace: jenkins
+  namespace: staging
```

Comment:
@123MwanjeMike any reason for changing the namespace? I think having jenkins in its own namespace makes resource management easier

Reply:
We don't have a jenkins namespace. It would be the only service there if one was created for it

Reply:
We can create it. [One if the files](https://github.com/airqo-platform/AirQo-api/blob/70e8ed8f621248f3559f222c14a590aa4e66dfe2/k8s/jenkins/jenkins-ns.yaml) you deleted contained the namespace configuration . 
I don't think being the only service has any draw backs because the aim is to separarte functionality.

Reply:
I think it would be better to have a cicd namespace instead so it may accommodate Jenkins (and other ci tools to be added soon). What’s your view on this @NoahNsimbe?

Reply:
yeah, that is cool @123MwanjeMike. Thanks